project/src/components/property-screen/form-property-screen.tsx

```diff
@@ -12,16 +14,41 @@ const STAR_TITLES: string[] = [
   'badly',
   'terribly',
 ];
+const MIN_REVIEW_LENGTH = 50;
+const MAX_REVIEW_LENGTH = 300;
+
 
 function FormPropertyScreen(): JSX.Element {
+  const isFormCleared = useAppSelector((state)=>state.isFormCleared);
+  useEffect(()=> {
+    if (!isFormCleared) {
+      store.dispatch(clearForm(true));
+    }
+  });
+  const isFormSend = useAppSelector((state)=>state.isFormSend);
   const [formData, setFormData] = useState<CommentData>({
     review: '',
     rating: '',
   });
 
-  const {id} = useParams();
-  const stringId = id as string;
+  const { id } = useParams();
+  let isReviewLengthCorrect = false;
+  let isRatingChecked = false;
 
+  if (
+    formData.review.length >= MIN_REVIEW_LENGTH &&
+    formData.review.length <= MAX_REVIEW_LENGTH
+  ) {
+    isReviewLengthCorrect = true;
+  } else {
+    isReviewLengthCorrect = false;
+  }
+
+  if (formData.rating === '') {
+    isRatingChecked = false;
+  } else {
+    isRatingChecked = true;
+  }
```

Comment:
Это пишется гораздо проще обычно

```suggestion
  const isReviewLengthCorrect = formData.review.length >= MIN_REVIEW_LENGTH && formData.review.length <= MAX_REVIEW_LENGTH
  const isRatingChecked = formData.rating !== '';
```